apps/akasha/src/components/post-page/post-page.tsx

```diff
@@ -131,11 +131,7 @@ const PostPage: React.FC<IPostPageProps & RootComponentProps> = props => {
   const isFollowing = followedProfiles.includes(entryData?.author?.ethAddress);
 
   const handleLoadMore = () => {
-    if (
-      reqComments.status === 'success' &&
-      reqComments.hasNextPage &&
-      loginState.currentUserCalled
-    ) {
+    if (reqComments.status === 'success' && reqComments.hasNextPage && loginState.fromCache) {
```

Comment:
kinda don't like checking for strings, this can be refactored into `reqComments.isSuccess`

Reply:
this is a list of available flags https://react-query.tanstack.com/reference/useQuery